Code/GraphMol/SmilesParse/CanonicalizeStereoGroups.h

```diff
@@ -0,0 +1,47 @@
+//
+//  Copyright (C) 2002-2021 Greg Landrum and other RDKit contributors
```

Comment:
Please just make this 2024. And it doesn't need to be my name

Reply:
ok